### PR TITLE
RHEV ISO fix

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 68a46f9695c9a14ae9b546c1c20549db4b99bd53:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/services/test_iso_service_catalogs.py [hunk 1]
```diff
@@ -104,7 +114,7 @@ def catalog():
 
 
 @pytest.yield_fixture(scope="function")
-def catalog_item(setup_provider, provider_crud, provider_type, provisioning, vm_name, dialog,
+def catalog_item(provider_init, provider_crud, provider_type, provisioning, vm_name, dialog,
```
Comment (psav) [inline cfme/tests/services/test_iso_service_catalogs.py]: We should be using the setup_provider fixture wherever possible. In 2116, we have created a separate module scoped setup_provider fixture I think, I am trying to remove the these custome provider_init functions throughout the code. Maybe we'll just wait for this one till 2116?

File: cfme/tests/services/test_iso_service_catalogs.py [hunk 2]
```diff
@@ -33,6 +34,9 @@ def pytest_generate_tests(metafunc):
             # No provisioning data available
             continue
 
+        if args['provider_type'] == "scvmm":
```
Comment (psav) [inline cfme/tests/services/test_iso_service_catalogs.py]: What about VMware? I don't think they support ISO either do they? So maybe we should flip it and say ```if not rhevm```

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 20bbefc553f9cc5a4746897fdfd20ff1bdaf264b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**